README.md

```diff
@@ -28,6 +28,26 @@ Mongodb
     - https://www.mongodb.com/blog/post/server-selection-next-generation-mongodb-drivers
     - http://docs.mongodb.org/manual/reference/read-preference
 
+## Usage
+
+
+### Connection Pools
+
+```elixir
+defmodule MongoPool do
+  use Mongo.Pool, name: __MODULE__, adapter: Mongo.Pool.Poolboy
+end
+
+# Starts a process on the module name MongoPool
+{:ok, _} = MongoPool.start_link(database: "test")
+
+# Gets an enumerable cursor for the results
+cursor = Mongo.find(MongoPool, "test-collection", %{})
+
+Enum.to_list cursor
+|> IO.inspect
```

Comment:
Sorry, missed this one. You need parenthesis for the function call otherwise it will be parsed as `Enum.to_list(cursor |> IO.inspect)`.